shared/chat/inbox/row/index.tsx

```diff
@@ -6,69 +6,47 @@ import SmallTeam from './small-team/container'
 import {BigTeamsLabel} from './big-teams-label'
 import {Box} from '../../../common-adapters'
 import {globalStyles, globalMargins, isMobile} from '../../../styles'
-import * as Types from '../../../constants/types/chat2'
-import * as TeamTypes from '../../../constants/types/teams'
-import * as RPCChatTypes from '../../../constants/types/rpc-chat-gen'
+import type * as Types from '../../../constants/types/chat2'
 
-type MakeRowOptions = {
```

Comment:
we don't need this 'merged' type, we already use a switch below so we can know the types